### ISSUE TITLE
0.9.0 release

### ISSUE THREAD

**Issue by rickysarraf** [stolowski/QComicBook#21]
Hi,

I see you have 0.9.0 marked in your changelog, back from 2011. But on release page on github, I do not see any 0.9.0 release. Nor do I see any reference to it elsewhere.

Is there a 0.9.0 release ? If so, where do I get it from ?

**Comment by rickysarraf** [comment]
In your changelog, you say

```
2011-05-30:  0.9.0   -- dramatic improvement in responsiveness and performance, thanks to delegating
	                    image transformations to separate thread.
	                 -- migrated internally to QGraphicsView
	                 -- implemented Zoom Lens.
	                 -- fixed memory leak when using small cursor.
	                 -- various bugfixes related to Continuous view mode.

```

which I think you meant is 2012. Please fix that and push the release.

**Comment by stolowski** [comment]
Apologies, I think this was due to the fact that I didn't push git tags. Anyway, it should be fixed now with the release of 0.9.1.
